.vscode/settings.json

```diff
@@ -6,4 +6,13 @@
     "esbonio.sphinx.confDir": "${workspaceFolder}/docs",
     "python.linting.mypyEnabled": true,
     "mypy.runUsingActiveInterpreter": true,
+    "python.testing.unittestArgs": [
+        "-v",
+        "-s",
+        "./tests",
+        "-p",
+        "test_*.py"
+    ],
+    "python.testing.pytestEnabled": false,
+    "python.testing.unittestEnabled": true,
```

Comment:
What do these settings do? (I tried googling but couldn't quite find what -v, -s, and -p do in this case)

Reply:
Ah sorry, sometimes (can't even tell you when) they get generated automatically when I run pre-commit, but seams to be a with my configs. I will remove the lines.

Reply:
Hahaha, vscode settings are mysterious to me as well :P - I only notice when they don't work